Source/Core/Common/CompatPatches.cpp

```diff
@@ -98,7 +98,7 @@ class ImportPatcher
       {
         for (auto name = names; name->u1.Function; name++)
         {
-          if (!IMAGE_SNAP_BY_ORDINAL(name->u1.Ordinal))
+          if ((name->u1.Ordinal & IMAGE_ORDINAL_FLAG64) == 0)
```

Comment:
I fail to see how this is simpler than the previous piece of code. Aside from the fact that this is already an obscure piece of code overall (that you don't see a whole lot otherwise.)

Normally, if macros are provided for value testing, they are provided for a reason and should be used...at least thats my opinion on this.